quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,200 @@
+Name:         postconfigure
+Namespace:    add-users-example
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+add-users.cli:
+----
+embed-server --std-out=echo  --server-config=standalone-openshift.xml
+batch
+
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity(identity=user1)
+/subsystem=elytron/filesystem-realm=KieFsRealm:set-password(identity=user1, clear={password="pass123*"})
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity-attribute(identity=user1, name=role, value=["kie-server","rest-all","admin","kiemgmt","Administrators","user"])
+
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity(identity=user2)
+/subsystem=elytron/filesystem-realm=KieFsRealm:set-password(identity=user2, clear={password="pass123*"})
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity-attribute(identity=user2, name=role, value=["kie-server","rest-all","admin","kiemgmt"])
+
+run-batch
+quit
+
+delayedpostconfigure.sh:
+----
+This empty file is needed until the following jira is not fixed:
+https://issues.redhat.com/browse/RHPAM-3665
+
+postconfigure.sh:
+----
+echo "******  RUNNING ADDITIONAL CONFIGURATIONS WITH JBOSS-CLI - ADDING EXTRA ELYTRON USERS TO KIE FS REALM **********"
+echo "trying to execute /opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli"
+/opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli
+echo "END - users added"
+
+Events:  <none>
+```
+
+
+With the config-map created, you can now proceed with the container configuration.
+In this example there is only one KIE Server running. The steps will be described by using the `operator` and 
+`application templates`.
+
+
+### Application Templates method:
+
+When using the `application templates` or an already running RHPAM instance, it turns to be a little easier to 
+configure than using `operators`.
+
+With the config map created, mount it as a `volume` on the desired Deployment Config:
+
+```bash
+$ oc set volumes dc/myapp-kieserver --add  \
+    --configmap-name=postconfigure \
+    --mount-path=/opt/eap/extensions \
+    --default-mode=0555
+```
+
+Note, if the target *dc* is other than `myapp-kieserver` remember to update it to fit your needs.
+This change will make the running KIE Server be redeployed, after it starts again you should see a message similar to:
+
+```bash
+******  RUNNING ADDITIONAL CONFIGURATIONS WITH JBOSS-CLI - ADDING EXTRA ELYTRON USERS TO KIE FS REALM **********
+trying to execute /opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli
+...
+some jboss logging output
+...
+The batch executed successfully
+15:33:07,651 INFO  [org.jboss.as] (MSC service thread 1-2) WFLYSRV0050: JBoss EAP 7.4.1.GA (WildFly Core 15.0.4.Final-redhat-00001) stopped in 40ms
+END - users added
+```
+
+### Operator method
+
+When using operator you need to edit the `kieconfig-<current-version>`, in this case, `7.12.0`.
+With the operator already running in the current namespace, follow the steps below:
```

Comment:
I think it needs to be made a bit clearer that the step must be run between subscribing tot he operator and deploying a KieApp.

Reply:
makes sense, added a link to our docs.